docs/design/fabric-v2+/interfaces.md

```diff
@@ -304,58 +241,117 @@ public int ecall_next_block(
         [user_check] uint8_t *block_bytes, uint32_t block_size);
 ```
 
-```c++
-// this secure session functions are defined in a secure channel module
-//included from secure_channel.edl
-
-public int session_setup(
-        [in] setup SessionSetup,
-        [out] SessionSetupResponse);
 
-public int session_close(
-        [in] close SessionClose
-        [out] SessionCloseResponse);
 
-public int session_request(
-        [in] request SessionRequest,
-        [out] SessionRequestResponse;
+Over the [Ledger Enclave - FPC Stub Secure Channel](interfaces.ecc-tlcc-channel.md),
+TLCC_Enclave also offers following RPC functionality to ECC, represented as
+protobuf request and response messages:
 ```
 
+```protobuf
+syntax = "proto3"
 
-Session requests are dispatched to the following interfaces:
-```c++
-// returns a cryptographic hash of the genesis block this tlcc enclave is initialized with.
-public hash get_channel_hash();
 
 // TODO update UMLs: these functions replace "check_ercc_commited_data"
 
-// verify state data
-public metadata get_state_metadata(
-        const char *namespace,
-        const char *key);
+// - verify state data
+
+//    public metadata get_state_metadata(
+//            const char *namespace,
+//            const char *key);
```

Comment:
I think this can go away

Reply:
well, it essentially illustrates the functional interface of RPC defined by the messages, that's why i retained it  (from shim and/or your original text). So at some level i thought it useful. but if you think it distracts or, worse, confuses, i'll remove it ...